components/financial_assistance/app/views/financial_assistance/incomes/_self_employed_income.html.erb

```diff
@@ -1,41 +1,78 @@
-<div id="<%= dom_id income %>" class="income">
-  <div class="display-income">
-    <div class="focus_effect_module">
-      <div class="row row-form-wrapper no-buffer row-height">
-        <div class="col-md-2 no-pd form-group-lg no-pd class-fa-household">
-          <label class="static_label label-floatlabel">NET AMOUNT</label>
-          <span class="field_value floatlabel form-control active-floatlabel" id="name"><%= number_to_currency(income.amount) %></span>
-        </div>
+<% if @bs4 %>
+  <div id="<%= dom_id income %>" class="income my-4">
+    <div class="display-income">
+      <div class="focus_effect_module">
+        <div class="d-flex col-sm income-form-row p-0 mb-2">
+          <div class="w-100 pr-3">
+            <label class="static_label label-floatlabel required" for="name-<%= income.amount %>" ><%= l10n("faa.other_incomes.amount") %></label>
+            <input class="fake-disabled-input w-100" id="name-<%= income.amount %>" disabled="true" value="<%= number_to_currency(income.amount) %>">
+          </div>
 
-        <div class="col-md-3 no-pd form-group-lg no-pd class-fa-household">
-          <label class="static_label label-floatlabel">HOW OFTEN</label>
-          <span class="field_value floatlabel form-control active-floatlabel" id="name"><%= frequency_kind_options.invert[income.frequency_kind] %></span>
-        </div>
+          <div class="w-100 pr-3">
+            <label class="static_label label-floatlabel required" for="name-<%= frequency_kind_options.invert[income.frequency_kind] %>-<%=income.id%>"><%= l10n("faa.other_incomes.frequency") %></label>
```

Comment:
legacy classes should be removed

Reply:
removed static_label and label-floatlabel throughout